src/ophys_etl/workflows/ophys_experiment.py

```diff
@@ -85,3 +91,114 @@ def from_id(
             session=session,
             specimen=specimen
         )
+
+    def get_ophys_experiment_motion_border(self,
+            session: Session) -> Dict:
+        """
+        Get motion border for an ophys experiment
+
+        Parameters
+        ----------
+        ophys_experiment_id
+            The ophys experiment id
+        session
+            The database session
+
+        Returns
+        -------
+        Dict[int]
+            A dictionary containing motion border data
+        """
+
+        workflow_step_run_id = get_latest_run(session,
+                                              WorkflowStepEnum.MOTION_CORRECTION,
+                                              WorkflowNameEnum.OPHYS_PROCESSING,
+                                              )
+        query = (
+            select(
+                MotionCorrectionRun,
+            )
+            .where(MotionCorrectionRun.workflow_step_run_id == workflow_step_run_id) 
+        )
+
+        result = session.execute(query).all()
+        motion_border = result[0][0]
+        return {
+            "x0": motion_border.max_correction_left,
+            "x1": motion_border.max_correction_right,
+            "y0": motion_border.max_correction_up,
+            "y1": motion_border.max_correction_down
+        }
+
+    def _generate_binary_mask(self, ophys_roi: OphysROI, ophys_roi_mask_values: List[OphysROIMaskValue]) -> List[List[bool]]:
+        """
+        Generate binary mask for an ROI
+
+        Parameters
+        ----------
+        ophys_roi
+            The ophys ROI
+        ophys_roi_mask_values
+            The ophys ROI mask values
+
+        Returns
+        -------
+        List[List[bool]]
+            A list of lists of booleans representing the binary mask
+        """
+        mask = np.zeros((ophys_roi.height, ophys_roi.width), dtype=bool)
+        for ophys_roi_mask_value in ophys_roi_mask_values:
+            mask[ophys_roi_mask_value.row_index, ophys_roi_mask_value.col_index] = True
+        return mask.tolist()
+    
+    def get_ophys_experiment_roi_metadata(
```

Comment:
See comment about converting this to a property and creating `session` rather than passing it.